src/components/Form.js

```diff
@@ -217,7 +216,18 @@ class Form extends React.Component {
     const disableContactInputs = !(acl.update_contacts || acl.send_application);
 
     return (
-      <div className="form">
+      <div>
+        {successMessages.map(successMessage => (
```

Comment:
are you duplicating error and success message on purpose ?

Reply:
I was trying to duplicate ActionButtons.